### ISSUE TITLE
Use blade instead of twig

### ISSUE THREAD

**Issue by bobmulder** [barryvdh/laravel-form-bridge#36]
Hi @barryvdh,

I love the approach of the Symfony Forms library, so I'm happy with your bridge! 

However, I don't feel that comfortable using a small piece of twig while Laravel uses blade. It feels overkill and I can imagine performance of the application won't get better since we have an extra render library.

I am wondering if we can make this library work with blade support, so we don't have to use twig anymore. I won't ask you to build this, but can you give me any directions? Should I lean on this library, or should I implement my own 'bridge' on Symfony Forms?

If you want to discuss this in more detail (in dutch :)), feel free to get in touch!

Regards, Bob

**Comment by barryvdh** [comment]
Yeah not sure, maybe we should see if it's actually slow? Because it just compiles to php anyway.
